src/main/java/core/basesyntax/service/ReadFromFileService.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ReadFromFileService {
```

Comment:
what is the meaning of this class in the context of the task?

if it's purpose is simply to check if we have written the correct output - let's make a private method with this logic in the test class